### PR TITLE
a new theme:axin,based on 'sexy'

### PR DESCRIPTION
**Pull request by songxinqiang**
Hey, I have created a new theme 'axin', it's based on the theme 'sexy'.:)
theme-axin

### CODE REVIEW

File: themes/axin/axin.theme.bash [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    RESET="\033[m"
+fi
+
+parse_git_dirty () {
+  [[ $(git status 2> /dev/null | tail -n1 | cut -c 1-17) != "nothing to commit" ]] && echo "*"
+}
+parse_git_branch () {
+  git branch --no-color 2> /dev/null | sed -e '/^[^*]/d' -e "s/* \(.*\)/\1$(parse_git_dirty)/"
+}
+
```
Comment (edubxb) [inline themes/axin/axin.theme.bash]: the functions:

    parse_git_dirty
    parse_git_branch

are not used, it is possible to remove it?

Also, you don't need to check with `git branch` to have the " on " string before the git information, use the environment var `SCM_THEME_PROMPT_PREFIX` to achieve this.

Comment (songxinqiang) [inline themes/axin/axin.theme.bash]: I have edit the theme.:)

**Comment by songxinqiang** [comment]
Hi , please merge the code.

**Comment by nwinkler** [comment]
Thanks!

**Pull request merged**